src/sentry/api/endpoints/relocations/index.py

```diff
@@ -164,14 +166,9 @@ def get(self, request: Request) -> Response:
 
         logger.info("relocations.index.get.start", extra={"caller": request.user.id})
 
-        # Non-superusers can only see their own relocations.
         queryset = Relocation.objects.all()
-        is_superuser = False
-        try:
-            is_superuser = SuperuserPermission().has_permission(request, None)
-        except Exception:
-            pass
```

Comment:
I refactored this to no longer throw, and we should be using `is_active_superuser` anyways